### PR TITLE
Fix 1.11 compile errors.

### PR DESCRIPTION
**Pull request by Natanel-Shitrit**

### CODE REVIEW

File: scripting/lilac.sp [hunk 1]
```diff
@@ -294,7 +294,7 @@ public void map_teleport(const char[] output, int caller, int activator, float d
 	playerinfo_time_teleported[activator] = GetGameTime();
 }
 
-public Action timer_welcome(Handle timer, int userid)
```
Comment (Natanel-Shitrit) [inline scripting/lilac.sp:297]: https://github.com/alliedmodders/sourcepawn/issues/613

File: scripting/lilac/lilac_aimbot.sp [hunk 2]
```diff
@@ -337,7 +337,7 @@ static void lilac_detected_aimbot(int client, float delta, float td, int flags)
 	}
 }
 
-public Action timer_decrement_aimbot(Handle timer, int userid)
```
Comment (Natanel-Shitrit) [inline scripting/lilac/lilac_aimbot.sp:340]: https://github.com/alliedmodders/sourcepawn/issues/613

File: scripting/lilac/lilac_aimlock.sp [hunk 3]
```diff
@@ -257,7 +257,7 @@ static bool lilac_is_player_in_aimlock_que(int client)
 			&& playerinfo_time_aimlock[client] > 1.0)); // Had one aimlock the past three minutes.
 }
 
-public Action timer_decrement_aimlock(Handle timer, int userid)
```
Comment (Natanel-Shitrit) [inline scripting/lilac/lilac_aimlock.sp:260]: https://github.com/alliedmodders/sourcepawn/issues/613

File: scripting/lilac/lilac_bhop.sp [hunk 4]
```diff
@@ -150,7 +150,7 @@ static void lilac_ban_bhop(int client)
 	lilac_ban_client(client, CHEAT_BHOP);
 }
 
-public Action timer_decrement_bhop(Handle timer, int userid)
```
Comment (Natanel-Shitrit) [inline scripting/lilac/lilac_bhop.sp:153]: https://github.com/alliedmodders/sourcepawn/issues/613

File: scripting/lilac/lilac_globals.sp [hunk 5]
```diff
@@ -198,4 +198,4 @@ bool playerinfo_banned_flags[MAXPLAYERS + 1][CHEAT_MAX];
 
 
 // Forward declare SourceIRC native so we can compile without it but still use it if present:
-native Function IRC_MsgFlaggedChannels(const char[] flag, const char[] format, any:...);
```
Comment (Natanel-Shitrit) [inline scripting/lilac/lilac_globals.sp:201]: old syntax.

**Comment by azalty** [comment]
You're quoting https://github.com/alliedmodders/sourcepawn/issues/613 which appears to be reversed.

It shouldn't break on compile anymore, so is this PR really needed?

**Comment by azalty** [comment]
You're quoting https://github.com/alliedmodders/sourcepawn/issues/613 which appears to be reversed.

It shouldn't break on compile anymore, so is this PR really needed?

**Comment by Natanel-Shitrit** [comment]
For now the "fix" is on the dev branch, I'm using latest compiler and it's still errors, i guess it's up to the repo owner to accept it or not, i don't mind.

**Comment by J-Tanzanite** [comment]
Oh, so in the 1.11 development version of Sourcemod, using `int` instead of `any` in timers causes errors?
Skimming through the discussion you link to, it seems like they decided not to do this?
Not gonna lie, I'm a little confused... @~@

I'll happily accept this PR if this change really does end up happening to SM, but I'll wait a bit and see if it becomes necessary first.
Still, thanks for the PR and the heads up :) :+1:

**Comment by Natanel-Shitrit** [comment]
> Oh, so in the 1.11 development version of Sourcemod, using `int` instead of `any` in timers causes errors?

In the `1.11 development version` this problem is fixed.

> Skimming through the discussion you link to, it seems like they decided not to do this?
> Not gonna lie, I'm a little confused... @~@

The change is already made in the main branch, latest complier errors. (i had a problem with my GitHub workflow that's pulling the latest compiler and compiles plugins, this plugin failed because the errors above)

> I'll happily accept this PR if this change really does end up happening to SM, but I'll wait a bit and see if it becomes necessary first.
> Still, thanks for the PR and the heads up :) 👍

You can accept and revert if they fix it in the future but you can let this boil until they push an update and reject it, I'm cool with both, or anything you wanna do.

**Comment by J-Tanzanite** [comment]
I have decided to merge this.
But I would prefer if this could be applied to the development branch.
Could you start a PR for that branch?
If not, I can do it manually :P 

Anyway, thanks for this PR, sorry for having been so slow to respond '^^

**Comment by J-Tanzanite** [comment]
Oh right, I forgot this PR existed, just fixed it myself in #150 
Closing as this is no longer needed, sorry for the delay and for having done it myself.

**Pull request closed without merging**